spec/factories/miq_group.rb

```diff
@@ -1,11 +1,20 @@
 FactoryGirl.define do
+  sequence(:miq_group_description) { |n| "Test Group #{seq_padded_for_sorting(n)}" }
```

Comment:
Oh interesting, it's just a method sort of...not tied to a column until you call generate.